pulsar-broker/src/main/java/com/yahoo/pulsar/broker/PulsarService.java

```diff
@@ -565,41 +567,56 @@ public MessagingServiceShutdownHook getShutdownService() {
     /**
      * Derive the host
      *
-     * @param isBindOnLocalhost
-     * @return
+     * @return Hostname or IP address the service binds on.
      */
     public static String host(ServiceConfiguration config) {
```

Comment:
Sure `getBindAddress()` sounds good, but we probably don't even need to have the static method, since we could just call `config.getBindAddress()` as it is. 

For rebasing, I will squash it anyway when merging. I don't expect particular conflicts with current master.